modules/op-creating-pipeline-templates-admin-console.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/working-with-pipelines-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="op-creating-pipeline-templates-admin-console_{context}"]
+= Creating pipeline templates in the Administrator perspective
+
+As a cluster administrator, you can create pipeline templates which developers can reuse when they create a pipeline on the cluster.
```

Comment:
Original description IMHO explained better why would somebody do it
> As a cluster administrator, to create and deploy an application from a Git repository, you can use custom pipeline templates that override the default pipeline templates provided by {pipelines-title}

Reply:
@mramendi +1 that's what I'm wondering, why can't they have both on a cluster?

Reply:
I guess that they can have both but our customers don't want those default templates. @vdemeester can you confirm?

Reply:
@ppitonak @vdemeester can you please confirm if there is anything here worth blocking the PR over, or can it be addressed in a future Jira, etc?

Reply:
I am actually not sure 😅 I tend to like @ppitonak suggestion better, but I don't have strong opinion 😓 and I don't think we need to block the PR on this 🙃

Reply:
Yup, let's not block this PR